web/src/components/JsonViewer/JsonViewer.tsx

```diff
@@ -42,24 +47,47 @@ const JsonViewer: React.FC<JsonViewerProps> = ({ data, collapsed }) => {
     [theme]
   );
 
+  const depth = React.useMemo(() => {
+    if (collapsed && !isExpanded) {
+      return false;
+    }
+    if (isExpanded) {
+      return 100;
+    }
+    return 1;
+  }, [collapsed, isExpanded]);
+
   const handleCopy = React.useCallback(copy => {
     copyTextToClipboard(JSON.stringify(copy.src, null, '\t'));
   }, []);
 
   return (
     <React.Suspense fallback={null}>
-      <ReactJSONView
-        src={data}
-        name={false}
-        theme="shapeshifter"
-        iconStyle="triangle"
-        displayObjectSize={false}
-        displayDataTypes={false}
-        collapsed={collapsed || 1}
-        style={jsonViewerStyle}
-        sortKeys
-        enableClipboard={handleCopy}
-      />
+      <Box position="relative">
```

Comment:
```suggestion
      <Box position="relative" width="100%">
```

I think we need full width to avoid issues with  Flex Parents

<img width="1355" alt="Screen Shot 2020-10-19 at 7 45 32 PM" src="https://user-images.githubusercontent.com/10436045/96481211-e8a26e00-1243-11eb-9e84-27f24030885c.png">